modules/starting-a-workbench.adoc

```diff
@@ -4,7 +4,7 @@
 = Starting a workbench
 
 [role='_abstract']
-You can manually start a data science project's workbench from the *Details* page for the project. By default, workbenches start immediately after you create them.
+You can manually start a data science project's workbench from the project details page. By default, workbenches start immediately after you create them.
```

Comment:
Looks like this functionality is on the Workbenches tab now, i.e.: 'You can manually start a data science project's workbench from the *Workbenches* tab for the project'.